### DISCUSSION TITLE
shen-ping/info/3C5G3xK91FP7Xg_RdO-du

### DISCUSSION THREAD

**Discussion by giscus** [wurencaideli/dumogu-ypwz-comment#3] [Announcements]
# shen-ping/info/3C5G3xK91FP7Xg_RdO-du

不是只有视频和音乐才带给人欢乐！还有一篇文字哟！学生时代所看过的短文还有谁记得呢？甚是怀念呢！

https://www.dumogu.top/shen-ping/info/3C5G3xK91FP7Xg_RdO-du

**Comment by wurencaideli** [comment]
这个就搞笑了
